boards/atsame54_xpro/src/pins.rs

```diff
@@ -0,0 +1,444 @@
+    // pin spi4_ss = b28,   // SPI_SS_A SERCOM4 Pad2
+    pin spi4_mosi = b27, // SPI_MOSI SERCOM4 Pad0
+    // pin spi4_miso = b29, // SPI_MISO SERCOM4 Pad3
+    pin spi4_sck = b26,  // SPI_SCK SERCOM4 Pad1
+
+    // Exension Header EXT2
+    pin pb00 = b0,  // ADC+
+    pin pa03 = a3,  // ADC-
+    pin pb01 = b1,  // GPIO1
+    pin pb06 = b6,  // GPIO2
+    pin pb14 = b14, // PWM+
+    pin pb15 = b15, // PWM-
+    pin pd00 = d0,  // IRQ/GPIO
+    pin pb02 = b2,  // SPI_SS_B/GPIO
+    pin spi6_ss = c6,    // SPI_SS_A SERCOM6 Pad2
+
+    // Exension Header EXT3
+    pin pc02 = c2,   // ADC+
+    pin pc03 = c3,   // ADC-
+    pin pc01 = c1,   // GPIO1
+    pin pc10 = c10,  // GPIO2
+    pin pd10 = d10,  // PWM+
+    pin pd11 = d11,  // PWM-
+    pin pc30 = c30,  // IRQ/GPIO
+    pin pc31 = c31,  // SPI_SS_B/GPIO
+    pin uart1_rx = c23,  // SERCOM5 Pad1
+    pin uart1_tx = c22,  // SERCOM5 Pad0
+    pin pc14 = c14,      // SPI_SS_A SERCOM6 Pad2
+
+    // SD/SDIO
+    pin sd_d0 = b18,
+    pin sd_d1 = b19,
+    pin sd_d2 = b20,
+    pin sd_d3 = b21,
+    pin sd_clk_i2s_sdo = a21,
+    pin sd_cmd_i2s_fs0 = a20,
+    pin sd_cd = d20,
+    pin sd_wp = d21,
+
+    // PCC Camera Connector
+    // pin pcc_vsync = a12,
+    // pin pcc_hsync = a13,
+    // pin pcc_pclk = a14,
+    // pin pcc_xclk = a15,
+    // pin pcc_dout7 = a23,
+    // pin pcc_dout6 = a22,
+    // pin pcc_dout5 = a21,
+    // pin pcc_dout4 = a20,
+    // pin pcc_dout3 = a19,
+    // pin pcc_dout2 = a18,
+    // pin pcc_dout1 = a17,
+    // pin pcc_dout0 = a16,
+    // pin pcc_dout9 = b15,
+    // pin pcc_dout8 = b14,
+    // pin pcc_reset = c12,
+    // pin pcc_pwdn = c11,
+
+    // Position Decoder Header
+    pin pdec_phase_a = c16,
+    pin pdec_phase_b = c17,
+    // pin pdec_index = c18,    // same as led
+
+    pin vbat = b3,
+
+    // ADC/DAC Header
+    pin adc_dac = a2,
+
+    // USB
+    pin vbus_detecion = c0,
+    pin usb_id = c19,
+    pin usb_dm = a24,
+    pin usb_dp = a25,
+
+    pin swd = a31,
+    pin swc = a30,
+    pin swo = b30,
+
+    // CAN
+    pin can_tx = b12,
+    pin can_rx = b13,
+    pin can_standby = c13,
+
+    // Ethernet
+    pin gtxck = a14,
+    pin gtxen = a17,
+    pin gtx0 = a18,
+    pin gtx1 = a19,
+    pin grxdv = c20,
+    pin grx0 = a13,
+    pin grx1 = a12,
+    pin grxer = a15,
+    pin gmdc = c11,
+    pin gmdio = c12,
+    pin ethernet_interrupt = d12,
+    pin ethernet_reset = c21,
+
+    // QSPI
+    pin flash_io0 = a8,
+    pin flash_io1 = a9,
+    pin flash_io2 = a10,
+    pin flash_io3 = a11,
+    pin flash_sck = b10,
+    pin flash_cs = b11,
+
+    // I²S
+    pin i2s_sck0 = b16,
+    pin i2s_mck0 = b17,
+    pin i2s_sdi = a22,
+    pin i2s_fs1 = a23,
+    pin i2s_sck1 = b28,
+    pin i2s_mck1 = b29,
+
+    // Data Gateway Interface
+    pin dgi_ss = d1,
+);
+
+impl Pins {
+    /// Split the device pins into subsets
+    pub fn split(self) -> Sets {
+        let analog = Analog {
+        //     a0: self.a0,
```

Comment:
I forget if I already asked about these commented out pins. Is this still WIP?

Reply:
The problem is that all of these pins are already there with other names. It's the same with the PCC pins. I didn't want to remove it, because I thought it might be useful to someone, but didn't know how to solve this better.
The current naming scheme is based on the board docs.

Reply:
I see. Then I guess this is good to go. Thanks again!